src/base/net/private/geoipdatabase.cpp

```diff
@@ -26,6 +26,8 @@
  * exception statement from your version.
  */
 
+#include <boost/detail/endian.hpp>
```

Comment:
Qt has Q_BYTE_ORDER for this purposes so I would use it instead of Boost.

Reply:
OK, changed.